control-plane/tests/test_cli_inspection.py

```
@@ -4035,6 +4251,79 @@ def test_cli_inspect_alert_detail_renders_alert_scoped_runtime_visibility(
             "analyst-004",
         )
 
+    def test_cli_inspect_alert_detail_classifies_unresolved_review_without_execution(
+        self,
+    ) -> None:
+        _, service, promoted_case, evidence_id, reviewed_at = self._build_phase19_in_scope_case()
```

Ruff (0.15.10)

[warning] Unpacked variable `evidence_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)